ios/ARTShadow.h

```diff
@@ -0,0 +1,12 @@
+//
```

Comment:
I feel like this comment shouldn't be here :P

Reply:
Updated with the comment used in all other files